content/academy/apify_platform/getting_started/apify_api.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Apify API
+description: Learn how to use the Apify API to programmatically call your actors, retrieve data stored on the platform, view actor logs, and more!
+menuWeight: 4
+paths:
+    - apify-platform/getting-started/apify-api
+---
+
+# [](#the-apify-api) The Apify API
+
+[Apify's API](https://docs.apify.com/api/v2#/reference) is your ticket to the Apify platform without even needing to access the [Apify Console](https://console.apify.com) web-interface. The API is organized around RESTful HTTP endpoints.
+
+In this lesson, we'll be learning how to use the Apify API to call an actor and view its results. We'll be using the actor we created in the previous lesson, so if you haven't already gotten that one set up, go ahead do that before moving forward if you'd like to follow along.
+
+## [](#finding-your-endpoint) Finding your endpoint
+
+Within one of your actors on the [Apify Console](https://console.apify.com) (we'll use the **adding-actor** from the previous lesson), click on the **API** button in the top right-hand corner:
+
+![The "API" button on an actor's page on the Apify Console]({{@asset apify_platform/getting_started/images/api-tab.webp}})
+
+You should see a long list of API endpoints that you can copy and paste elsewhere, or even test right within the **API** modal. Go ahead and copy the endpoint labeled **Run actor synchronously and get dataset items**. It should look something like this:
+
+```text
+https://api.apify.com/v2/acts/YOUR_USERNAME~adding-actor/run-sync?token=YOUR_TOKEN
+```
+
+> In this lesson, we'll only be focusing on this one endpoint, as it is the most popularly used one; however, don't let this limit your curiosity! Take a look at the other endpoints in the **API** window to learn about everything you can do to your actor programmatically.
+
+Now, let's move over to [Insomnia]({{@link tools/insomnia.md}}) in order to prepare the request, then send it.
```

Comment:
Might be worth showing a quick example for CURL as well. Plus mention that they can use Postman or their favorite HTTP client.

Reply:
done